drake/CMakeLists.txt

```diff
@@ -326,6 +322,12 @@ if (MATLAB_FOUND)
     PROPERTIES TIMEOUT 300)
 endif()
 
+option(BUILD_DOCUMENTATION "Build Doxygen and Sphinx documentation." ON)
```

Comment:
I can switch switch it to default `OFF`. You do not have to compile the whole of drake if you do not want to. Just build the documentation CMake target.